tests/test_crash_similarity.py

```diff
@@ -85,6 +86,39 @@ def test_train_model(self):
         self.assertEqual(20, resp.iter)
         self.assertEqual(101, len(resp.wv.vocab))
 
+    def test_wmdistance_non_zero_distance(self):
```

Comment:
Nit: move this right after `test_order_similarity`, this way we keep all model-related tests together.